_includes/doc-widget.html

```diff
@@ -25,7 +30,13 @@
   {% if item.icon == "fa-code" %}
     {% assign a_href = master_blob %}
   {% elsif item.icon == "fa-code-fork" %}
+    {% if lang != "" %}
+      <!-- skip "Edit This Page" button on translated pages, link to Transifex only -->
+      {% continue %}
+    {% endif %}
```

Comment:
But our English doc pages have `lang: en`. Won't this skip the "Edit This Page" button on the English pages too?

Reply:
No, `lang` is a variable with URL prefix, set to empty if `page.lang` is either empty or `en`.